docs/man-openmpi/man3/MPI_T_category_get_events.3.rst

```diff
@@ -0,0 +1,50 @@
+.. _mpi_t_category_get_events:
+
+
+MPI_T_category_get_events
+=========================
+
+.. include_body
+
+:ref:`MPI_T_category_get_events` |mdash| Query which events are in a
+category
+
+
+SYNTAX
+------
+
+
+C Syntax
+^^^^^^^^
+
+.. code-block:: c
+
+   #include <mpi.h>
+
+   int MPI_T_category_get_events(int cat_index, int len, int indices[])
```

Comment:
We'll need to update this PR or #13179, depending on who merges first (i.e., generate the bindings that appear in the man pages).